src/ApiService/ApiService/onefuzzlib/PoolOperations.cs

```diff
@@ -87,8 +87,8 @@ public PoolOperations(ILogTracer log, IOnefuzzContext context)
         return await _context.Queue.QueueObject(GetPoolQueue(pool.PoolId), workSet, StorageType.Corpus);
     }
 
-    public IAsyncEnumerable<Pool> GetByClientId(Guid clientId) {
-        return QueryAsync(filter: $"client_id eq '{clientId}'");
+    public IAsyncEnumerable<Pool> GetByObjectId(Guid objectId) {
+        return QueryAsync(filter: $"object_id eq '{objectId}'");
```

Comment:
Won't this need a column rename?

Reply:
no need, the `client_id` was not used before. in the worst case, we will have entities with both the client_id and the object_id